src/main/java/com/worktracker/model/dto/ReportByProjectResponseDTO.java

```diff
@@ -0,0 +1,57 @@
+package com.worktracker.model.dto;
+
+import com.worktracker.model.TypeTask;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportByProjectResponseDTO {
+
+    List<ReportUserHoursDTO> users = new ArrayList<>();
+    private TypeTask typeTask;
+    private String taskName;
+    private Double totalHoursWork = 0.0;
+
+    public Double getTotalHoursWork() {
+        this.calculateTotalHours();
+        return totalHoursWork;
+    }
+
+    public TypeTask getTypeTask() {
+        return typeTask;
+    }
+
+    public void setTypeTask(TypeTask typeTask) {
+        this.typeTask = typeTask;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public void setTaskName(String taskName) {
+        this.taskName = taskName;
+    }
+
+    public List<ReportUserHoursDTO> getUsers() {
+        return this.users;
+    }
+
+    private void calculateTotalHours() {
+        double sum = 0;
+        for (ReportUserHoursDTO u : users) {
```

Comment:
This is OK, but lets try making it with stream for practice.
The flow will be something like: map to hours -> sum terminating operation.